extensions/amp-story/1.0/amp-story.css

```diff
@@ -394,6 +394,11 @@ amp-story-grid-layer a * {
   pointer-events: none !important;
 }
 
+/** Ensures only the visible parts of the reactions are clickable. */
+amp-story-grid-layer .i-amphtml-story-reaction {
+  pointer-events: none !important;
+}
```

Comment:
In some layouts the reactions were taking all the screen space (minus margins) so on the empty spaces the clicks would not trigger page changes. With this, only the visible reaction component overrides the clicks, and the rest of the amp-story-page receives clicks as expected

Reply:
That looks like a short term hack to fix a broader Reaction component positioning issue. Can we remove this for now, file a bug, and solve it in another PR?

Having the reaction component filling the space will cause more issues, like the inability to use flex CSS rules to position it in a corner or on a side. This is out of scope for this PR though.

Reply:
I think it has to do with the template system, since usually other elements are not clickable but this one is. However, we can work on it on another PR, so I removed it from here